modules/apps/translation/translation-web/src/main/resources/META-INF/resources/import_translation_results.jsp

```diff
@@ -54,121 +54,163 @@ renderResponse.setTitle(LanguageUtil.get(resourceBundle, "import-translation"));
+					<div>
+						<div class="panel panel-secondary" role="tablist">
+							<button aria-expanded="false" class="<%= importTranslationResultsErrors ? "collapsed" : StringPool.BLANK %> btn btn-unstyled collapse-icon collapse-icon-middle panel-header panel-header-link" role="tab" type="button">
+								<liferay-util:whitespace-remover>
+									<div class="h4 mb-0 text-success">
+										<span class="mr-2">
+											<clay:icon
+												symbol="check-circle-full"
+											/>
+										</span>
+										<%= importTranslationResultsDisplayContext.getSuccessMessageLabel(locale) %>
 									</div>
+								</liferay-util:whitespace-remover>
+
+								<span class="collapse-icon-closed">
+									<clay:icon
+										symbol="angle-right"
+									/>
+								</span>
+								<span class="collapse-icon-open">
+									<clay:icon
+										symbol="angle-down"
+									/>
+								</span>
+							</button>
+
+							<div class="<%= importTranslationResultsErrors ? "collapse" : StringPool.BLANK %> panel-collapse" role="tabpanel">
+								<div class="panel-body">
+									<ul class="list-group list-group-no-bordered mb-0">
+
+										<%
+										for (String successMessage : importTranslationResultsDisplayContext.getSuccessMessages()) {
+										%>
+
+											<li class="align-items-center list-group-item">
+												<div class="list-group-title"><%= successMessage %></div>
+											</li>
+
+										<%
+										}
+										%>
+
+									</ul>
 								</div>
-							</li>
-
-						<%
-						}
-						%>
-
-					</ul>
+							</div>
+						</div>
+					</div>
 				</c:if>
 
 				<c:if test="<%= importTranslationResultsDisplayContext.getFailureMessagesCount() > 0 %>">
-					<h4 class="text-danger">
-						<span class="mr-2">
-							<clay:icon
-								symbol="exclamation-full"
-							/>
-						</span>
-
-						<liferay-ui:message arguments="<%= importTranslationResultsDisplayContext.getFailureMessagesCount() %>" key='<%= (importTranslationResultsDisplayContext.getFailureMessagesCount() == 1) ? "x-error" : "x-errors" %>' />
-
-						<small><liferay-ui:message key="some-files-could-not-be-published-check-them-and-upload-another-file" /></small>
-					</h4>
-
-					<ul class="list-group list-group-no-bordered">
-
-						<%
-						Map<String, String> failureMessages = importTranslationResultsDisplayContext.getFailureMessages();
-
-						for (Map.Entry<String, String> entry : failureMessages.entrySet()) {
-						%>
-
-							<li class="align-items-center list-group-item list-group-item-flex">
-								<div class="autofit-col autofit-col-expand">
-									<div class="list-group-title"><%= entry.getKey() %></div>
-									<div class="text-danger"><%= entry.getValue() %></div>
-								</div>
-
-								<div class="autofit-col text-danger text-right">
-									<div class="list-group-icon">
-										<clay:icon
-											symbol="exclamation-full"
+					<clay:sheet
+						size=""
```

Comment:
Is it possible to just omit this property?

Reply:
Not because if we omitted it the [<clay:sheet](https://github.com/liferay-lima/liferay-portal/blob/eeb0e25f2b869d02fde0ca8ee0aaea5c396c5089/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/SheetTag.java#L55-L57) will be size `lg` as default. We need it fluid.

Reply:
Yep, it's weird. My knowledge about Java Tags isn't good. 
If someone has a better option to apply the same container width on the panel I'm happy to apply it.

Reply:
Hey @boton,

> Not because if we omitted it the [<clay:sheet](https://github.com/liferay-lima/liferay-portal/blob/eeb0e25f2b869d02fde0ca8ee0aaea5c396c5089/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/SheetTag.java#L55-L57) will be size lg as default. We need it fluid.

If I understand correctly you'd like the `size` attribute to have the value `"fluid"`, is that correct?

If that's the case, you could add that logic [here](https://github.com/liferay/liferay-portal/blob/6b59cd0f086b56ef57502ab1a365e70d84eb5c73/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/SheetTag.java#L55-L57) or change the [default value](https://github.com/liferay/liferay-portal/blob/6b59cd0f086b56ef57502ab1a365e70d84eb5c73/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/SheetTag.java#L64) of the `size` attribute.

Let me know what you think.

Reply:
Hey @boton thanks for the additional information

> The problem is I don't want to add a size class I try to omit it but if it is null the default value size lg is applied.

I understand. If `size="ful"` works for you; I think that's much better than `size="<%= StringPool.BLANK %>"`. Let me know if that works. Thanks.

Reply:
I added the `full` size https://github.com/liferay-frontend/liferay-portal/blob/3879153bda2df2617d3442a3a366819a8a00fbe0/modules/apps/translation/translation-web/src/main/resources/META-INF/resources/import_translation_results.jsp#L134 I think that is the better option too.

Reply:
@boton seems good to me, thanks!